.cirrus/slack-notification.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -euo pipefail
+
+curl -X POST https://slack.com/api/chat.postMessage \
+  -H "Authorization: Bearer ${SLACK_TOKEN}" \
+  -H 'Content-type: application/json; charset=utf-8' \
+  --data-binary @- <<EOF
+{
+  "channel": "team-sonarlint-vscode",
```

Comment:
`squad-ide-vscode` or `squad-ide-vscode-bots` ?

Reply:
@sophio-japharidze-sonarsource I copy pasted it from the release workflow (to always have the "correct" one), maybe this is incorrect as well. I can change both if you like 😄

Reply:
I guess it also got outdated after the renaming, we haven't released yet since then to see it fail :D

Reply:
Done!